integration-cli/docker_cli_cp_to_container_test.go

```diff
@@ -21,124 +17,6 @@ import (
 // 3. DST parent directory must exist.
 // 4. If DST exists as a file, it must not end with a trailing separator.
 
-// First get these easy error cases out of the way.
-
-// Test for error when SRC does not exist.
-func (s *DockerSuite) TestCpToErrSrcNotExists(c *check.C) {
```

Comment:
Added in https://github.com/docker/cli/pull/907 as `TestRunCopyToContainerSourceDoesNotExist`